bioimageio/core/prediction_pipeline/_prediction_pipeline.py

```diff
@@ -139,25 +139,12 @@ def output_axes(self):
     def input_shape(self):
         return self._input_shape
 
+    # todo: separate preprocessing/actual forward/postprocessing
```

Comment:
I agree and would also suggest that we at least try to come to some kind of understanding what different terms imply:

* `preprocess`, `postprocess`: obvious
* `forward`: run only the network
* `predict`: run full pipeline (preprocessing + network + postprocessing)

Reply:
Next PR